samples/matter/light_bulb/boards/nrf21540dk_nrf52840.overlay

```diff
@@ -55,3 +55,23 @@
 	};
 
 };
+
+/* Disable unused peripherals to reduce power consumption */
```

Comment:
Minor: I think it doesn't matter, as light bulb is not battery supplied by design